src/plugins/virtualbox/vmitem.cpp

```diff
@@ -0,0 +1,80 @@
+// albert - a simple application launcher for linux
+// Copyright (C) 2016 Martin Buergmann
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "vmitem.h"
+
+#include <QProcess>
+
+/** ***************************************************************************/
+QString VirtualBox::VMItem::iconPath_;
```

Comment:
Sollte in einen [anonymen namespace](http://www.comeaucomputing.com/techtalk/#nostatic) um der global namespace pollution entgegen zu wirken.